helm/nessie/README.md.gotmpl

```diff
@@ -129,6 +129,90 @@ This is broadly following the example from https://kubernetes.io/docs/tasks/acce
 * Use the IP from the above output and add it to `/etc/hosts` via `echo "192.168.49.2 chart-example.local" | sudo tee /etc/hosts`
 * Verify that `curl chart-example.local` works
 
+### OpenTelemetry Collector with Minikube
+
+* Start Minikube cluster: `minikube start`
+* Create K8s Namespace: `kubectl create namespace nessie-ns`
+* Install cert-manager:
+
+```bash
+kubectl apply -f https://github.com/cert-manager/cert-manager/releases/download/v1.11.0/cert-manager.yaml
+```
+
+* Install Jaeger Operator:
+
+```bash
+kubectl create namespace observability
+kubectl apply -f https://github.com/jaegertracing/jaeger-operator/releases/download/v1.42.0/jaeger-operator.yaml -n observability
+```
+
+If the above command fails with "failed to call webhook [...] connection refused", then cert-manager
+was not yet ready. Wait a few seconds and try again.
+
+* Create a Jaeger instance in Nessie's namespace:
+
+```bash
+kubectl apply -n nessie-ns -f - <<EOF
+apiVersion: jaegertracing.io/v1
+kind: Jaeger
+metadata:
+  name: jaeger
+EOF
+```
+
+If the above command fails with "failed to call webhook [...] connection refused", then the Jaeger
+Operator was not yet ready. Wait a few seconds and try again.
+
+* Install Nessie Helm chart with OpenTelemetry Collector enabled:
+
+```bash
+helm install nessie -n nessie-ns helm/nessie \
+  --set tracing.enabled=true \
+  --set tracing.endpoint=http://jaeger-collector:4317
```

Comment:
Would it work with the `otlp/2` protocol (default port `55680`)?

Reply:
I confess I don't know, I am going to run a few tests now to investigate.

Reply:
Here is what I found:

According to this issue: https://github.com/open-telemetry/opentelemetry-specification/issues/1148, it seems that in its early stages, OpenTelemetry chose to use ports 55680 and 55681 for OTLP/gRPC and OTLP/HTTP respectively.

However, as the discussion in the issue reveals, these choices weren't good ones, because these ports are in the ephemeral range, and consequently, are not suitable to be used as service ports.

So, they asked IANA to reserve instead two other ports: 4317 and 4318, the common ports that we use today for OTLP/gRPC and OTLP/HTTP respectively.

If you look at [the specs in 2020](https://github.com/open-telemetry/opentelemetry-specification/blob/cb25fe94d8d9378c04ee37d7802e7552cf5b6cc0/specification/protocol/exporter.md#configuration-options), the default endpoint for OTLP/gRPC was `localhost:55680`.

But if you look at [the specs now](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/protocol/exporter.md), the default endpoint for OTLP/gRPC was changed to `http://localhost:4317`.

As a conclusion, I don't think there is any other alternate port that we should be supporting, only 4317 is nowadays being used for OTLP/gRPC.

Reply:
nice :+1: thanks for the detailed explanation!